db/migrations/003-t&c.ts

```diff
@@ -0,0 +1,11 @@
+import { DataTypes, QueryInterface } from 'sequelize'
+
+export const up = async (queryInterface: QueryInterface) => {
+  await queryInterface.addColumn(`Groups`, 'termsAndConditionsAcceptedAt', {
+    allowNull: false,
+    type: DataTypes.DATE,
+  })
```

Comment:
**problem:** this needs to be a **User** property, not a **Group** property.

Reply:
Hmm, what if one user creates multiple groups?

Reply:
That's fine. In that case they have already accepted the Terms & Conditions. They apply to humans, not to groups. Groups are legally represented by humans, so the humans have to accept the terms.

Reply:
Thanks for explanation! I'll work on change and create new PR